plexupdate-core

```diff
@@ -51,7 +51,7 @@ getPlexToken() {
 			fi
 		done
 		while true; do
-			read -e -p "PlexPass Password: " -i "$PASS" PASS
+			read -e -s -p "PlexPass Password: " -i "$PASS" PASS
```

Comment:
I originally kept password echo on since plexupdate is meant to still be accessible to non-technical people, and they often get confused when no output appears while typing. (And bash still doesn't provide a way to replace characters with `*` or something similar.)

I'm ok with trying it and seeing if we get any issues filed about it, but I also view it as unnecessary in this case, so I'll leave the decision up to you.